kolibri/plugins/coach/assets/src/views/exams/CreateExamPage/CreateExamPreview.vue

```diff
@@ -0,0 +1,333 @@
+                  :questionNumberWithinExercise="questionIndex"
+                  :totalFromExercise="availableExamQuestionSources.length"
+                  :isSelected="isSelected(question.itemId, exercise.exercise_id)"
+                  :exerciseName="getExerciseName(exercise.exercise_id)"
+                  :isCoachContent="Boolean(numCoachContents(exercise))"
+                  @click="goToQuestion(question.itemId, exercise.exercise_id)"
+                />
+              </ol>
+            </div>
+          </KGridItem>
+          <KGridItem
+            sizes="3, 5, 8"
+            :style="{ maxHeight: `${maxHeight}px` }"
+            class="o-y-auto"
+          >
+            <ContentRenderer
+              v-if="content && itemId"
+              :id="content.id"
+              ref="contentRenderer"
+              :kind="content.kind"
+              :files="content.files"
+              :contentId="content.content_id"
+              :available="content.available"
+              :extraFields="content.extra_fields"
+              :itemId="itemId"
+              :assessment="true"
+              :allowHints="false"
+              :showCorrectAnswer="true"
+              :interactive="false"
+            />
+          </KGridItem>
+        </KGrid>
+      </div>
+    </transition>
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import find from 'lodash/find';
+  import { ContentNodeResource } from 'kolibri.resources';
+  import { createQuestionList, selectQuestionFromExercise } from 'kolibri.utils.exams';
+  import KModal from 'kolibri.coreVue.components.KModal';
+  import ContentRenderer from 'kolibri.coreVue.components.ContentRenderer';
+  import KButton from 'kolibri.coreVue.components.KButton';
+  import KRadioButton from 'kolibri.coreVue.components.KRadioButton';
+  import KGrid from 'kolibri.coreVue.components.KGrid';
+  import KGridItem from 'kolibri.coreVue.components.KGridItem';
+  import KCircularLoader from 'kolibri.coreVue.components.KCircularLoader';
+  import responsiveWindow from 'kolibri.coreVue.mixins.responsiveWindow';
+  import debounce from 'lodash/debounce';
+  import AssessmentQuestionListItem from '../AssessmentQuestionListItem';
+
+  export default {
+    name: 'CreateExamPreview',
+    $trs: {
+      title: 'Select questions',
+      back: 'Go back',
+      finish: 'Finish',
+      numQuestions: '{num} {num, plural, one {question} other {questions}}',
+      exercise: 'Exercise { num }',
+      randomize: 'Give me a different set of questions',
+      questionOrder: 'Question order',
+      questions: 'Questions',
+      randomizedLabel: 'Randomized',
+      randomizedDescription:
+        'The questions in this quiz will be in a random order for each learner',
+      flexibleLabel: 'Flexible',
+      flexibleDescription:
+        'You can order the quiz questions however you want. All learners will see this version',
```

Comment:
Seeing 2 different strings for this - just verifying that this is the right one?